BingWallpaper/Downloader.cs

```diff
@@ -28,13 +28,25 @@ public Downloader(IWebAccessor webAccessor)
             State = LoadState();
         }
 
-        public void Run()
+        public async void Run(int refrashTime)
         {
-            if (_task != null)
+            if (refrashTime > 0)
```

Comment:
I don't think it's a good idea to use `refreshTime == 0` to represent a force reload. The `Run` method here is to start a background thread which periodically check for updates and reload the wallpaper. For one time reload, we should cancel the current task and call `DoWork` directly in  `ReflashPicture` method.